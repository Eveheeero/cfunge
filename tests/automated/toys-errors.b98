"SYOT"4(v ; Test error handling for TOYS ;
        >   02- #vA na"A :DAB"  >:#,_v
v                >  na"A :DOOG" >:#,_>

>   0001-01-00  #vC na"C :DAB"  >:#,_v
v                >  na"C :DOOG" >:#,_>

>   0001-01-00  #vK na"K :DAB"  >:#,_v
v                >  na"K :DOOG" >:#,_>

>   0001-01-00  #vM na"M :DAB"  >:#,_v
v                >  na"M :DOOG" >:#,_>

>   0001-01-00  #vS na"S :DAB"  >:#,_v
v                >  na"S :DOOG" >:#,_>

>   0001-01-00  #vV na"V :DAB"  >:#,_v
v                >  na"V :DOOG" >:#,_>

@