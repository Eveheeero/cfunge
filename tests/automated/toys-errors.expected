GOOD: A
GOOD: C
GOOD: K
GOOD: M
GOOD: S
GOOD: V
